content/src/ui/Modal.Reminders.json

```diff
@@ -0,0 +1,46 @@
+{
+  "en": {
+    "title": "Reminders",
+    "actions": {
+      "sessionReminders": "Session reminders",
+      "practiceReminders": "Practice reminders"
+    },
+    "sessionRemindersText": "Reminds you about upcoming sessions 10 minutes before they start.",
+    "weekday": "Weekday",
+    "time": "Time",
+    "intervals": {
+      "daily": "Every day",
+      "monday": "Monday",
+      "tuseday": "Tuseday",
+      "wednesday": "Wednesday",
+      "thursday": "Thursday",
+      "friday": "Friday",
+      "saturday": "Saturday",
+      "sunday": "Sunday"
+    },
+    "updateButton": "Update reminders"
+  },
+  "pt": {},
+  "sv": {
+    "title": "Notiser",
+    "actions": {
+      "sessionReminders": "Notiser sessioner",
+      "practiceReminders": "Notiser övning"
```

Comment:
Translated these to "Sessionspåminnelser" and "Övningspåminnelser" in NotificationChannels. We should align them, or maybe even re-use?